### ISSUE TITLE
[bug]: (hugo) Many design options are missing

### ISSUE THREAD

**Issue by BenJamesBen** [freesewing/freesewing#3738]
### What seems to be the problem? 🤔

In the current `develop` lab development environment, Hugo is missing many design options seen in lab.freesewing.dev. Missing options include Chest Ease and Length Bonus.
![Screenshot 2023-04-02 at 5 49 13 PM](https://user-images.githubusercontent.com/109869956/229388977-93c22e7e-6391-433c-a089-bbb76e34ec11.png)

### Design / Plugin / Package 🧐

designs/hugo

### Are you a FreeSewing patron? 😃

No, I am not 😞

**Comment by BenJamesBen** [comment]
In `designs/hugo/src/options.mjs`:
```diff
-export const chestEase = { pct: 8, min: 4, max: 20 }
-export const hipsEase = { pct: 12, min: 4, max: 20 }
-export const cuffEase = { pct: 20, min: 10, max: 50 }
-export const lengthBonus = { pct: 10, min: 0, max: 20 }
-export const sleeveLengthBonus = { pct: 2, min: 0, max: 10 }
-export const ribbingStretch = { pct: 5, min: 0, max: 10 }
-export const ribbingHeight = { pct: 10, min: 4, max: 20 }
+export const chestEase = { pct: 8, min: 4, max: 20, menu: 'fit' }
+export const hipsEase = { pct: 12, min: 4, max: 20, menu: 'fit' }
+export const cuffEase = { pct: 20, min: 10, max: 50, menu: 'fit' }
+export const lengthBonus = { pct: 10, min: 0, max: 20, menu: 'style' }
+export const sleeveLengthBonus = { pct: 2, min: 0, max: 10, menu: 'style' }
+export const ribbingStretch = { pct: 5, min: 0, max: 10, menu: 'fit' }
+export const ribbingHeight = { pct: 10, min: 4, max: 20, menu: 'style' }
```

**Comment by bobgeorgethe3rd** [comment]
closed as off #3811
